chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/pendingListing/reviewer/ChplNumberReviewer.java

```diff
@@ -135,28 +150,24 @@ public void review(final PendingCertifiedProductDTO listing) {
 
                     if (!developer.getDeveloperCode().equals(developerCode)) {
                         listing.getErrorMessages()
-                        .add("The developer code '" + developerCode
-                                + "' does not match the assigned developer code for "
-                                + listing.getDeveloperName() + ": '" + developer.getDeveloperCode() + "'.");
+                                .add("The developer code '" + developerCode
+                                        + "' does not match the assigned developer code for "
+                                        + listing.getDeveloperName() + ": '" + developer.getDeveloperCode() + "'.");
                     }
                     if (certificationBody != null) {
-                        DeveloperACBMapDTO mapping = developerDao.getTransparencyMapping(developer.getId(),
-                                certificationBody.getId());
-                        if (mapping != null) {
-                            // check transparency attestation and url for
-                            // warnings
-                            if ((mapping.getTransparencyAttestation() == null
-                                    && listing.getTransparencyAttestation() != null)
-                                    || (mapping.getTransparencyAttestation() != null
-                                    && listing.getTransparencyAttestation() == null)
-                                    || (mapping.getTransparencyAttestation() != null
-                                    && !mapping.getTransparencyAttestation()
-                                    .equals(listing.getTransparencyAttestation()))) {
-                                listing.getWarningMessages().add(msgUtil.getMessage("transparencyAttestation.save"));
+                        if (!ff4j.check(FeatureList.EFFECTIVE_RULE_DATE_PLUS_ONE_WEEK)) {
+                            DeveloperACBMapDTO mapping = developerDao.getTransparencyMapping(developer.getId(),
+                                    certificationBody.getId());
+                            if (mapping != null) {
+                                // check transparency attestation and url for warnings
+                                if (!areTransparencyAttestationsEqual(mapping.getTransparencyAttestation(),
```

Comment:
In my experience the general consensus is to use isBooleanThing whether single or plural (isBooleanThings) but I rather like how you've broken convention here. Sticking it to the man :-)